DEPLOYMENT.md

```diff
@@ -0,0 +1,67 @@
+# Deployment
+
+## Requirements
+
+[Heroku cli](https://devcenter.heroku.com/articles/heroku-cli)
+
+## Deploy
+
+Clone the Heroku repo, if you haven't done so already:
+
+```bash
+$ heroku git:clone -a scopa-server
+$ cd scopa-server
+```
+
+Make some changes to the code you just cloned and deploy them to Heroku using Git.
+
+```bash
+$ git add .
+$ git commit -am "make it better"
+$ git push heroku-scopa-server HEAD:main
+```
+
+## Logs
+
+To check Heroku live logs, do:
+
+```bash
+heroku --app scopa-server logs --tail
+```
+
+## Setup
+
+Below are described the steps taken to setup Scopa on Heroku.
+
+### Server
+
+```bash
+$ heroku create scopa-server --remote heroku-scopa-server
+
+$ heroku buildpacks:add -a scopa-server heroku/nodejs
+$ heroku buildpacks:add -a scopa-server heroku-community/multi-procfile
+
+$ heroku config:set -a scopa-server PROCFILE=packages/server/Procfile
+$ heroku config:set -a scopa-server HTTP_ORIGIN=https://scopa-client.herokuapp.com
```

Comment:
do we need those variables in the server btw? HTTP_ORIGIN and SOCKET? Is it allright that are both pointing to the same url? I don't think this is right, isn't it? I know it's just documentation, but just asking